flask_discord_interactions/discord.py

```diff
@@ -615,7 +621,7 @@ def set_route_async(self, route, app=None):
         if app is None:
             app = self.app
 
-        if aiohttp == None:
+        if aiohttp is None:
```

Comment:
This one can stay? Comparing with `== None` is generally not a good idea, despite probably not mattering here.

Reply:
Yep, this is fine.